ios/Runner/Sites.swift

```diff
@@ -85,9 +85,9 @@ class SiteUpdater: NSObject, FlutterStreamHandler {
   private var notification: Any?
   public var startFunc: (() -> Void)?
   private var configFd: Int32? = nil
-  private var configObserver: DispatchSourceFileSystemObject? = nil
+  private var configObserver: (any DispatchSourceFileSystemObject)? = nil
 
-  init(messenger: FlutterBinaryMessenger, site: Site) {
+  init(messenger: any FlutterBinaryMessenger, site: Site) {
```

Comment:
Just curious, was this migration manual or automated?

Reply:
Manually clicking "fix" in XCode, not sure how to make it more like `eslint . --fix` where they're all applied at once